Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-
+gemspec
 gem 'pry'
```

Comment:
They are debug helpers. They aren't even development dependencies, since the test suite and other development code doesn't use it.

So leaving them here is perfectly acceptable.